test/app/gh-4235-dont-show-idle-in-fiber-info.test.lua

```diff
@@ -0,0 +1,44 @@
+test_run = require('test_run').new()
+netbox = require('net.box')
+fiber = require('fiber')
+
+test_run:cmd('setopt delimiter ";"')
+function count_idle()
+    local idle = 0
+    for fid, info in pairs(fiber.info()) do
+        if info.backtrace == nil then
+            idle = idle + 1
+        end
+    end
+    return idle
+end;
+
+cnt = 0;
+idle = 0;
+done = false;
+function yielder()
+    cnt = cnt + 1
+    while not done do
+        fiber.yield()
+    end
+    cnt = cnt - 1
+end;
+test_run:cmd('setopt delimiter ""');
+
+box.schema.user.grant('guest', 'execute', 'universe')
+conn = netbox.connect(box.cfg.listen)
+
+prev_idle = count_idle()
+
+n = 100
+for i = 1, n do conn:call('yielder', nil, {is_async = true}) end
```

Comment:
Could you please add comment explaining what's going on here. E.g. why do you need netbox? I suspect to generate fibers, but why don't you use fiber.create? Test doesn't seem obvious from the first sight...

Reply:
Added explanatory comment